index.html

```diff
@@ -0,0 +1,448 @@
+                        <p><a href="https://www.nytimes.com/2021/02/18/style/blue-light-glasses.html?action=click&algo=bandit-all-surfaces_desk_filter&block=editors_picks_recirc&fellback=false&imp_id=127613580&impression_id=fa6ae0e2-7431-11eb-acb8-b3f144742d83&index=2&pgtype=Article&region=ccolumn&req_id=544755693&surface=home-featured&variant=3_bandit-all-surfaces_desk_filter_dedup&action=click&module=editorContent&pgtype=Article&region=CompanionColumn&contentCollection=Trending">I Wanted To Love Her, Not Save Her</a></p>
+                    </section>
+                </aside>
+            </div>
+            <div class="contentSecTwo">
+                <section class="conSecTwo">
+                    <p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.</p>
+                    <p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+                    <h3>'As Big as It Gets'</h3>
+                    <p>In our own universe, it would serve as a window into the forces operating at energies forever beyond the reach of particle accelerators on Earth and yield new insights into gravity itself. Dr. Kovac’s ripples would be the first direct observation of gravitational waves, which, according to Einstein’s theory of general relativity, should ruffle space-time.</p>
+                    <p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”</p>
+                    <p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”</p>
+                    <p>The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.</p>
+                    <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+                </section>
+            </div>
+        </div>
+        <div class="midDiv">
+            <section class="midDivSec">
+                <article class="firstArt">
+                  <h3>The Theory of Inflation</h3>
```

Comment:
The class `firstArt` was never given any properties in your CSS. Please remove such unused class declarations from your code since they make the code base harder to maintain.